.github/workflows/integrationTest.yml

```diff
@@ -14,18 +14,73 @@ env:
   PASSPHRASE: ${{ secrets.PASSPHRASE }}
   GPG_KEY_NAME: ${{ secrets.GPG_KEY_NAME }}
   GPG_TTY: $(tty)
+  ECR_INTEGRATION_TEST_REPO: "cwagent-opensource-integration-test"
```

Comment:
I think that this needs to be abstracted like everything else so that we can run this from our forks with some flexibility.

Reply:
I'm fine with being abstract though; but since it is a environment variable `ECR_INTEGRATION_TEST_REPO` so it can be changed anytime depends on the developer's opinion.